qutip/solver/floquet.py

```diff
@@ -180,7 +195,13 @@ def state(self, t, data=False):
         else:
             return self._as_ketlist(states_mat)
 
-    def from_floquet_basis(self, floquet_basis, t=0):
+    QobjOrData = TypeVar("QobjOrData", Qobj, Data)
```

Comment:
`QobjOrData` seems common enough that it could migrate to the qutip typing module.